io/net/bridge.py

```diff
@@ -92,6 +95,9 @@ def test_bridge_delete(self):
         Set to original state
         '''
         self.check_failure('ip link del dev %s' % self.bridge_interface)
+        cmd = "rm -rf /etc/sysconfig/network-scripts/ifcfg-%s" \
+              % self.bridge_interface
+        process.system(cmd, shell=True)
```

Comment:
What is the reason to do this in test instead of utils ?

Reply:
@narasimhan-v this one is for remove cfg file for bridge interface.after link delete also if we do ifup of thta interface name its coming back.

Reply:
i can say use `os.remove` @narasimhan-v this we are in utils it just delete a file

Reply:
@smruti77 Did we try to add this to utils ? Would be better if handled via utils

Reply:
@PraveenPenguin I was talking about avocado network utils

Reply:
may we can add in genio utils like generic to remove as I did not feel wider use case for n/w utils either as I say just we can use `os.remove` from test

Reply:
ok then i will use os.remove

Reply:
@PraveenPenguin @smruti77 I am confused here.

Why should network configure utils not handle it ? How is it handled in other tests ?

Reply:
@narasimhan-v i am saying it just removing a file  , it can be part of utils but adding one line u feel we need to be part of utils as other case we need backup and we restore it from back but here we need to remove interface file